### PR TITLE
update HelloSignClient for issue #67

### PR DESCRIPTION
**Pull request by radhack**

### CODE REVIEW

File: src/main/java/com/hellosign/sdk/HelloSignClient.java [hunk 1]
```diff
@@ -1001,8 +1001,8 @@ public ApiApp updateApiApp(ApiApp app) throws HelloSignException {
         if (!app.hasClientId()) {
             throw new HelloSignException("Cannot update an ApiApp without a client ID. Create one first!");
         }
-        String url = API_APP_URI + "/" + app.getClientId();
-        return new ApiApp(httpClient.withAuth(auth).withPostFields(app.getPostFields()).put(url).asJson());
+        String url = BASE_URI + API_APP_URI + "/" + app.getClientId();
```
Comment (cmpaul) [inline src/main/java/com/hellosign/sdk/HelloSignClient.java]: @radhack I notice that other ApiApp methods are missing the BASE_URI, can you add there as well?

Comment (radhack) [reply src/main/java/com/hellosign/sdk/HelloSignClient.java]: @cmpaul done sir!

**Comment by cmpaul** [comment]
@radhack Can you remove the version bump and re-open this PR against the `develop` branch? I'll bump the version and deploy once I merge to master.

**Comment by cmpaul** [comment]
Re-opening.

**Pull request merged**